app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -367,6 +370,11 @@ def secure_attributes
         end
       end
 
+      def user_has_invalid_scope
+        return unless @page.scoped?
+        (@page.scoped_user & current_alchemy_user.alchemy_roles).empty?
```

Comment:
It we should rename the page method that returns the roles allowed to edit. See comment below